google/oauth2/_reauth_async.py

```diff
@@ -299,6 +302,11 @@ async def refresh_grant(
             == reauth._REAUTH_NEEDED_ERROR_RAPT_REQUIRED
         )
     ):
+        if not enable_reauth_refresh:
+            raise exceptions.RefreshError(
+                "Reauthentication is needed. Please run `gcloud auth application-default login` to reauthenticate."
```

Comment:
The command should be "gcloud auth login --update-adc"

Reply:
updated